docs/azure/ai/get-started-azure-openai.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Get started with the .NET and Azure Open AI
+description: Get started on how to use the Azure OpenAI with a gpt-35-turbo model, from a simple .NET console application. Get a hiking recommendation from the AI model. It consists of a simple console application, running locally, that will send request to an Azure OpenAI service deployed in your Azure subscription.
+ms.date: 02/20/2024
+ms.topic: get-started
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: fboucher
+ms.author: frbouche
+# CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact to learn from the sample code.
+---
+
+This sample demonstrates how to use the Azure OpenAI with a `gpt-35-turbo` model, from a simple .NET 8.0 console application. It consists of a simple console application, running locally, that will send request to an Azure OpenAI service deployed in your Azure subscription. Everything will be deployed automatically using the Azure Developer CLI.
+
+## Requirements
+
+- .NET 8.0 SDK - [Install the .NET 8.0 SDK](https://dotnet.microsoft.com/download/dotnet/8.0)
+- An Azure subscription - [Create one for free](https://azure.microsoft.com/free/)
+- Azure Developer CLI - [Install or update the Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
+
```

Comment:
Let's make the pre-reqs an include file. Add an `include` folder under this directory and add everything under the prerequisites header to it. Then reference that.

Reply:
Yes, good idea.
I started doing this also on the code side.